docs/conf.py

```diff
@@ -260,7 +272,7 @@ def __getattr__(cls, name):
 # (source start file, name, description, authors, manual section).
 man_pages = [
     ('index', 'cookiecutter', u'cookiecutter Documentation',
-     [u'Audrey Roy'], 1)
+     [u'Audrey Roy and cookiecutter community'], 1)
```

Comment:
Let's not change this yet and create an issue to update the author fields in `setup.py` as well?

Reply:
Cookiecutter moved to org and most last commits made by org. this is no point to discuss here community work should be mentioned here.

Reply:
```suggestion
     [u'Audrey Roy and Cookiecutter community'], 1)
```
This is a small enough change to sneak in here and I'm more than happy to credit the community. Just fix the capitalization 😃

Reply:
Resolved in 54f2efc